Src/Support/Google.Apis.Auth/OAuth2/UserCredential.cs

```diff
@@ -57,12 +62,21 @@ public TokenResponse Token
         /// <param name="token">An initial token for the user.</param>
         public UserCredential(IAuthorizationCodeFlow flow, string userId, TokenResponse token)
```

Comment:
Could I suggest that this constructor delegates to the other one instead? Typically it's simpler to reason about constructors where all constructors just delegate, except one that does all the work. So this would become `: this(flow, userId, token, null)`.